cpp/log/cluster_state_controller-inl.h

```diff
@@ -170,11 +180,62 @@ void ClusterStateController<Logged>::OnServingSthUpdated(
     LOG(WARNING) << "Cluster has no Serving STH!";
     actual_serving_sth_.reset();
   } else {
+    // TODO(alcutter): Validate STH and verify consistency with whatever we've
+    // already got locally.
+    if (!update.handle_.Entry().has_timestamp() ||
+        update.handle_.Entry().timestamp() == 0) {
```

Comment:
Ah, just off the dodgy half-hearted initialisation code we had which shoved in an empty file for `serving_sth`